core/tests/protractor_utils/forms.js

```diff
@@ -624,15 +624,19 @@ var CodeMirrorChecker = function(elem) {
    *  - 'checked': true or false, whether the line has been checked
    */
   var _compareTextFromLine = function(
-      currentLineNumber, scrollTo, compareDict) {
+      currentLineNumber, scrollTo, Pane, compareDict) {
+    /* eslint-disable */
```

Comment:
Why are we disabling eslint here, can you please explain?

Reply:
The below like has quotes (") which were caught by linter Thats why i kept it such

Reply:
Removed eslint used escape quotes